assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -84,8 +83,10 @@ module netcdf_utilities_mod
 interface nc_get_global_attribute
    module procedure nc_get_global_char_att
    module procedure nc_get_global_int_att
-   module procedure nc_get_global_real_att
-   module procedure nc_get_global_real_array_att
+   module procedure nc_get_global_float_att
+   module procedure nc_get_global_double_att
+   module procedure nc_get_global_float_array_att
+   module procedure nc_get_global_double_array_att
```

Comment:
The AMSUA netCDF files that is the CF-compliant version of the HDF-EOS file has global attributes that are 'float' and 'double'. Since the netCDF reader is a clone of the HDF reader, the variables were declared identically to the provided HDF example. 64-bit reals as well as 32-bit reals are needed, so there needed to be more routines in the interface.